contributors/devel/help-wanted.md

```diff
@@ -2,12 +2,91 @@
 
 Items marked with the `help wanted` label need to ensure that they are:
 
-- Sufficiently actionable: clear description of what should be done
-- Tractable for new/casual contributors: there is documentation how that type of change should be made
-- Goldilocks priority: Not too high that a core contributor should do it, but not too low that it isn't useful enough for a core contributor to spend time to review it, answer questions, help get it into a release, etc.
-- Up to date: Often these issues become obsolete and have already been done, are no longer desirable, no longer make sense, change in priority, change in difficulty, etc.
+- **Low Barrier to Entry**<br/>
+  It should be tractable for casual contributors. Documentation on how that type of change should be made should already exist.
+
+- **Clear Task**<br/>
+  The task is agreed upon and does not require further discussions
+  in the community. Call out if that area of code is untested and requires new fixtures.
+
+  API / CLI behavior is decided and included in the OP issue, for example: _"The new command syntax is `svcat unbind NAME [--orphan] [--timeout 5m]`"_, with expected validations called out.
+
+- **Goldilocks priority**<br/>
+  Not too high that a core contributor should do it, but not too low that it
+  isn't useful enough for a core contributor to spend time to review it, answer questions, help get it into a release, etc.
+
+- **Up-To-Date**<br/>
+  Often these issues become obsolete and have already been done, are no longer desired,
+  no longer make sense, have changed priority or difficulty , etc.
 
 Related commands:
 
 - `/help` : adds the `help wanted` label to an issue
-- `/remove-help` : removes the `help wanted` label from an issue
\ No newline at end of file
+- `/remove-help` : removes the `help wanted` label from an issue
+
+# Good First Issue
+
+Items marked with the `good first issue` label are intended for _new contributors_.
+These make it clear to new contributors that they are welcome, valued, and provide
+a clear starting point for joining the project. After a contributor has successfully
+completed 1-2 `good first issue`'s, they should be ready to move on to `help wanted` items, saving remaining `good first issue`'s for other new contributors.
+
+Please keep an eye out for pull requests from new contributors and help
+move it through our processes. New contributors shouldn't be left to decipher prow
+commands, build flakes, find an approver, or ping for reviews.
+
+These items need to ensure that they follow the guidelines for `help wanted` labels (above)
+in addition to meeting the following criteria:
+
+- **No Barrier to Entry**<br/>
+  The task is something that a new contributor can tackle without
+  advanced setup, or domain knowledge.
+
+- **Solution Explained**<br/>
+  The recommended solution is clearly described in the issue.
```

Comment:
This is a fair point but I'm just wondering if bugs also come under this?

If an issue is a bug, is the task of finding _why_  it occurs and _how_ to solve it, the responsibility of the new contributor or the experienced contributor who added the label?
Does the experienced contributor need to figure out the solution before adding the label or can they help the new contributor find the reason behind the bug and how to solve it?

Reply:
Both, I think!
In any case, previous research on the bug should be presented. If a solution exists, go ahead and give it.
If there's not a known solution for the bug, then I think the person applying the `good-first-issue` label (or someone also close to the issue) should be prepared to guide and mentor.

Reply:
I just realized that I didn't make any changes based on this feedback. Personally, I wouldn't apply a "good first issue" label to a bug that I didn't already have a solution in mind for.